catalyst_voices/lib/widgets/menu/voices_wallet_tile.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:catalyst_cardano/catalyst_cardano.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:flutter/material.dart';
+
+/// A replacement for the [ListTile] with customized
+/// styling that displays a [CardanoWallet].
+class VoicesWalletTile extends StatelessWidget {
+  /// base64 encoded image representing the wallet extension.
+  final String? icon;
```

Comment:
just from looking at `icon` i would not be able to tell this should be `base64`. How about renaming property to smth else or introduce some classes structure for image/icon source? Then you could have smth. like 
`final Base64Image? icon`.

Other problem here is that it's just a string an we could as well pass `url` without any warnings

Open question, it may require more work and should be done in separate PR.

Reply:
I renamed it to iconSrc, technically it can be URL or base64 encoded PNG. In either case, `Image.network()` can handle both.

Reply:
<img width="833" alt="Screenshot 2024-09-03 at 14 37 23" src="https://github.com/user-attachments/assets/b9438faa-8c0d-41e9-8116-22659c0b17ac">